src/Api/Controllers/AccountsController.cs

```diff
@@ -708,5 +713,33 @@ public async Task PutTaxInfo([FromBody]TaxInfoUpdateRequestModel model)
             };
             await _paymentService.SaveTaxInfoAsync(user, taxInfo);
         }
+
+        [HttpDelete("sso/{organizationId}")]
+        public async Task DeleteSsoUser(string organizationId)
+        {
+            var userId = _userService.GetProperUserId(User);
+            if (!userId.HasValue)
+            {
+                throw new NotFoundException();
+            }
+
+            var ssoUser = new SsoUser()
+            {
+                UserId = userId.Value,
+                OrganizationId = new Guid(organizationId),
+            };
+
+            await _organizationService.DeleteSsoUserAsync(ssoUser);
+        }
+
+        [HttpGet("sso/user-identifier")]
+        public async Task<string> GetSsoUserIdentifier(string authorizeUrl)
+        {
+            var user = await _userService.GetUserByPrincipalAsync(User);
+            var token = await _userService.GenerateSignInTokenAsync(user, TokenPurposes.LinkSso);
+            var bytes = Encoding.UTF8.GetBytes($"{user.Id.ToString()},{token}");
```

Comment:
You don't need the `.ToString()` when interpolating, it's called implicitly by the interpreter. Keeps things easy to read and succinct

Reply:
TIL & fixed